### PR TITLE
Set-Retweet: Convert to APIv2

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

File: BluebirdPS/public/api_v2/Set-Retweet.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        [Parameter(Mandatory)]
+        [string]$Id,
+        [Parameter(ParameterSetName='Retweet')]
+        [switch]$Retweet,
+        [Parameter(ParameterSetName='Unretweet')]
+        [switch]$Unretweet
+    )
+
+    if ($PSCmdlet.ParameterSetName -eq 'Retweet') {
+        $body = @{
```
Comment (thedavecarroll) [inline BluebirdPS/public/api_v2/Set-Retweet.ps1:14]: Please use PascalCase for properties, parameters, and variables (obviously not API parameters).

`$Body`

File: BluebirdPS/public/api_v2/Set-Retweet.ps1 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            tweet_id = $Id
+        }
+        $Request = [TwitterRequest]@{
+            HttpMethod = 'POST'
+            Endpoint = 'https://api.twitter.com/2/users/{0}/retweets' -f $BluebirdPSConfiguration.AuthUserId
+            body = ($body | Convertto-json -Depth 10 -Compress)
```
Comment (thedavecarroll) [inline BluebirdPS/public/api_v2/Set-Retweet.ps1:20]: Please use PascalCase for properties, parameters, and variables.

`Body = ($Body`

File: BluebirdPS/public/api_v2/Set-Retweet.ps1 [hunk 3]
```diff
@@ -0,0 +1,30 @@
+function Set-Retweet {
```
Comment (thedavecarroll) [inline BluebirdPS/public/api_v2/Set-Retweet.ps1:1]: Please move this file to `public/api_v2/tweets`

**Review by thedavecarroll — CHANGES_REQUESTED** [review]
Just a couple things.

Please use PascalCase for properties, parameters, and variables (obviously not API parameters).
Please use the public/api_v2/tweets/ folder.

File: BluebirdPS/public/api_v2/Set-Retweet.ps1 (hunk 1 above)
Comment (alagoutte) [reply BluebirdPS/public/api_v2/Set-Retweet.ps1]: Done

File: BluebirdPS/public/api_v2/Set-Retweet.ps1 (hunk 3 above)
Comment (alagoutte) [reply BluebirdPS/public/api_v2/Set-Retweet.ps1]: Oups, fixed !

**Pull request merged**